examples/resolved_cfd-dem/sphere_carreau_with_sharp_interface/sphere_carreau_with_sharp_interface.prm

```diff
@@ -0,0 +1,204 @@
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = viscous
+end
+
+# --------------------------------------------------
+# Post-processing
+#---------------------------------------------------
+subsection post-processing
+	set calculate apparent viscosity = true
+	set verbosity = verbose
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+set number of fluids = 1
+  subsection fluid 0
+   	set rheological model	= carreau
+
+    subsection non newtonian
+    	subsection carreau
+	    	set viscosity_0    = 0.063403
+	    	set viscosity_inf  = 0
+	    	set lambda	   = 10
+	    	set a	           = 2.0
+	    	set n 		   = 0.5
+	    end
+    end
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 2,1,1 : -18,-15,-15 : 42,15,15 : true
+    set initial refinement   = 4
+end
+
+subsection  box refinement
+    set initial refinement   = 3
+    subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 1,1,1: -2,-2,-2 : 6,2,2 : true
+    set initial refinement   = 0
+    end
+end
+
+
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+set number                  = 5
+    subsection bc 0
+	set id = 0
+        set type              = function
+        subsection u
+            set Function expression = 1
+        end
+        subsection v
+            set Function expression = 0
+        end
+        subsection w
+            set Function expression = 0
+        end
+    end
+    subsection bc 1
+	set id = 2
+        set type              = slip
+    end    
+    subsection bc 2
+	set id = 3
+        set type              = slip
+    end
+    subsection bc 3
+	set id = 4
+        set type              = slip
+    end
+    subsection bc 4
+	set id = 5
+        set type              = slip
+    end
+end
+
+# --------------------------------------------------
+# IB particules
+#---------------------------------------------------
+subsection particles
+	set number of particles = 1
+	set stencil order = 2
+	set length ratio  = 1
+	set refine mesh inside radius factor = 0.85
+	set refine mesh outside radius factor = 1.3
+	set initial refinement = 2
+	set integrate motion = false
+	set assemble Navier-Stokes inside particles = false    
+	subsection particle info 0
+	subsection position
+	    set Function expression =0;0;0
+	end
+	    set pressure x =0.00001
+	    set pressure y =0.00001
+	    set pressure z =0.00001
+	    set radius = 0.5
```

Comment:
I thought this option was not required anymore?

Reply:
I had no idea, I will remove them and see if it still works.